addons/account/models/account_report.py

```diff
@@ -175,8 +175,15 @@ def copy(self, default=None):
         code_mapping = {}
         for line in self.line_ids.filtered(lambda x: not x.parent_id):
             line._copy_hierarchy(code_mapping, report=self, copied_report=copied_report)
+        for column in self.column_ids:
+            column.copy({'report_id': copied_report.id})
```

Comment:
Rebasing your branch on 16.0 (I had to, because of recent template changes), I can't duplicate any report :o Can you have a look ?